bigquery_etl/udf/publish_udfs.py

```diff
@@ -134,6 +136,14 @@ def publish_udf(
             fr'library = "gs://{gcs_bucket}/{gcs_path}\1"', definition
         )
 
+        # add UDF descriptions
+        if raw_udf.filepath not in SKIP:
+            escaped_description = json.dumps(str(raw_udf.description))
+            query = OPTIONS_RE.sub(f"OPTIONS(description={escaped_description},", query)
```

Comment:
JavaScript UDFs already have some `OPTIONS` for importing external scripts, descriptions need to be added to the existing `OPTIONS`